scripts/core/editor3/actions/toolbar.jsx

```diff
@@ -84,11 +84,11 @@ export function removeLink() {
  * @description Displays the external upload dialog and returns the insert images
  * action.
```

Comment:
Would be cool to document this new parameter up here...

Reply:
done